api/provider/aws/common.go

```diff
@@ -82,7 +82,7 @@ func lookupTaskDefinitionARNFromDeployID(store tag.Store, deployID string) (stri
 func lookupEntityEnvironmentID(store tag.Store, entityType, entityID string) (string, error) {
 	tags, err := store.SelectByTypeAndID(entityType, entityID)
 	if err != nil {
-		return "", err
+		return "ts", err
```

Comment:
This seems curious to me. I didn't see any references to the string `"ts"` for each reference I could find for this function. Is this string being explicitly tested somewhere that I'm not seeing?

Reply:
Good catch. Hmm I don't recall adding this. I don't think it's necessary.